contracts/core/lib/CoreState.sol

```diff
@@ -134,35 +128,6 @@ contract CoreState {
         return state.factories;
     }
 
-    /**
-     * Return boolean indicating if address is valid Set.
-     *
-     * @param  _set           Set address
-     * @return bool           Boolean indicating if valid Set
-     */
-    function validSets(
```

Comment:
This needs to be in Core for issue and redeem. How are we supposed to know which factory to query?